src/BiDirectionalNodeImp.h

```diff
@@ -0,0 +1,51 @@
+//
+// Created by shvmo on 02/05/2020.
+//
+
+#ifndef DATASTRUCTURESWET1_BIDIRECTIONALNODEIMP_H
+#define DATASTRUCTURESWET1_BIDIRECTIONALNODEIMP_H
+
+#include "BiDirectionalNode.h"
```

Comment:
please remove

Reply:
Why not let it sit there? Again that'd what ifndef is for.